packages/wayland_utils.rb

```diff
@@ -1,21 +1,20 @@
 require 'buildsystems/meson'
 
-class Wayland_info < Meson
+class Wayland_utils < Meson
   description 'wayland-info is a utility for displaying information about the Wayland protocols supported by a Wayland compositor.'
```

Comment:
Update description?

Reply:
`wayland-info` is actually the ONLY binary provided by the `wayland_utils` package, so we don't need to update the description.
```
cat manifest/x86_64/w/wayland_utils.filelist
/usr/local/bin/wayland-info
/usr/local/share/man/man1/wayland-info.1.zst
```

Reply:
Rename the package?